app/assets/stylesheets/_users.scss

```diff
@@ -0,0 +1,115 @@
+// Place all the styles related to the users controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+$light-grey: lighten(#b2beb5, 10%);
```

Comment:
Color literals like `#b2beb5` should only be used in variable declarations; they should be referred to via variable everywhere else.